test_resources/version_with_dataclasses.py

```diff
@@ -14,6 +14,11 @@ class User:
     name: str
 
 
+@dataclass
+class DataclassWithOptional:
+    x: Optional[int]
```

Comment:
You might test more complicated scenarios I believe. Like `x: Optional[List[Optional[int]]]`

Reply:
Good suggestion, I'll add more complicated scenarios.

Reply:
Good suggestion, I'll add more complicated scenarios.